modules/openy_features/openy_block/modules/openy_block_branch_amenities/src/Plugin/Field/FieldFormatter/BranchAmenitiesFormatter.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Drupal\openy_block_branch_amenities\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FormatterBase;
+use Drupal\Core\Field\FieldItemListInterface;
+
+/**
+ * Plugin implementation of the 'openy_branch_amenities' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "openy_branch_amenities",
+ *   label = @Translation("Display Amenities for Current Branch Page"),
+ *   field_types = {
+ *     "boolean",
```

Comment:
@podarok Is it really applicable to `boolean` and `link` field types?

Reply:
Wait... the formatter applied to the block's field_branch_am which is boolean, then `entity_reference` and `link` are odd.

Reply:
Fixed https://github.com/ymcatwincities/openy/pull/2205/commits/fab3ceeac8bf10c3e34a96273c91142dd763d325